### DISCUSSION TITLE
Is there any way to get the video frame number every 200ms?

### DISCUSSION THREAD

**Discussion by currysesame** [cookpete/react-player#1778] [Q&A]
Be sure to [search for your issue](https://github.com/CookPete/react-player/issues) before opening a new one.

#### Current Behavior

Is there any way to let me get the video frame number for each 200ms?
Already test the getCurrentTime, but it renew the number per second.

#### Expected Behavior

Return the video frame number when the video is playing for each 200ms. 
If the video stopped, return the same number would be great.

Tried to use getCurrentTime in this file: react-player/src/players/FilePlayer.js to get the current time, but that is not fast enough.
Tried this one: https://github.com/CookPete/react-player/issues/163.
But the progress.played updated per second. 

How can I renew the getCurrentTime/progress.played faster ?

#### Environment
- URL attempting to play: mp4 local file
- Browser: Chrome

**Comment by hoetmaaiers** [comment]
Also curious, @currysesame do you have any new knowledge since Nov 9, 2022?

**Comment by currysesame** [comment]
Yes. I had used another method to work around.
And it's fine now. I can play video and show something at the same time.

**Comment by hoetmaaiers** [comment]
Would you mind sharing the method you use for capturing the current frame number?

**Comment by Spritan** [comment]
@currysesame can you shre the work around, i need some way to get frame number faster than once per second
